src/persist_data/persist_books.rb

```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+module PersistBooks
+  SOURCE = './DB/books.json'.freeze
+
+  def self.read_from_file
+    deserialized_books = JSON.parse(File.read(SOURCE))
```

Comment:
- [ ] Kindly handle the case when the file is missing for (books,people, rentals), you can handle this by adding the following line
```ruby
 return [] unless File.exists?(SOURCE)
```

Reply:
Done! 😄